lab_1/start.py

```diff
@@ -37,4 +37,4 @@
 
     RESULT = top_words

```

Comment:
please, put in RESULT concordance/adjacent words and on 40 write down the expected concordance/adjacents words